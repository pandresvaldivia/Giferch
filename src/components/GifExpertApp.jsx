import AddCategory from './AddCategory';
import GifGrid from './GifGrid';
import Header from './Header';

const GifExpertApp = () => {
	return (
		<>
			<Header />
			<main className="px-8 py-6 max-w-screen-xl mx-auto">
				<AddCategory />
				<GifGrid amount={12} />
			</main>
		</>
	);
};

export default GifExpertApp;
