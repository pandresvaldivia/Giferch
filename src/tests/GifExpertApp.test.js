import { shallow } from 'enzyme';

import GifExpertApp from '../components/GifExpertApp';

describe('These are the tests for the GifExpertApp component', () => {
	test('Should return the initial state of the GifExpertApp component', () => {
		const wrapper = shallow(<GifExpertApp />);
		expect(wrapper).toMatchSnapshot();
	});
});
