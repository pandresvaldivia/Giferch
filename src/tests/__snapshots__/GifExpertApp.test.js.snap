// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`These are the tests for the GifExpertApp component Should return the initial state of the GifExpertApp component 1`] = `
<Fragment>
  <Header />
  <main
    className="px-8 py-6 max-w-screen-xl mx-auto"
  >
    <SearchCategory
      category="Rick and Morty"
      setCategory={[Function]}
    />
    <GifGrid
      amount={16}
      category="Rick and Morty"
    />
  </main>
</Fragment>
`;
